test/GoodGhosting_No_External_Pool_Deposits.test.js

```diff
@@ -179,20 +180,53 @@ contract("GoodGhosting_No_External_Pool_Deposit", (accounts) => {
 
                 truffleAssert.eventEmitted(result, "Withdrawal", async (ev) => {
                     console.log(`player${i+1} withdraw amount: ${ev.amount.toString()}`);
-                    const eventAmount = new BN(ev.amount.toString());
                     if (GoodGhostingArtifact === GoodGhostingPolygon) {
                         const playersMaticBalance = new BN(await rewardToken.methods.balanceOf(player).call({ from: admin }));
-                        return ev.player === player && playerPayment.eq(eventAmount) && playersMaticBalance.eq(playerMaticBalanceBeforeWithdraw);
+                        success = success
```

Comment:
why this success = success was it used for debugging?

Reply:
okay although I think it was working when I tested but yeah no worries good to merge